docs/self-managed/setup/guides/connect-to-an-oidc-provider.md

```diff
@@ -108,6 +103,10 @@ global:
 </TabItem>
 </Tabs>
 
+:::warning
+Once set, your initial claim name and value cannot be updated using environment or Helm values, and must be changed directly in the database.
```

Comment:
Non-blocking suggestion (nbs):

Once set, you cannot update your initial claim name and value using environment or Helm values. You must change these values directly in the database.